src/resources/MachineLearning/SemanticEncoderConfiguration/SemanticEncoderConfiguration.ts

```diff
@@ -0,0 +1,15 @@
+import API from '../../../APICore.js';
+import Resource from '../../Resource.js';
+import {
+    SemanticEncoderDocumentGroupPreview,
+    SemanticEncoderDocumentGroupPreviewParams,
+} from './SemanticEncoderConfigurationInterfaces.js';
+
+export default class SemanticEncoderConfiguration extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/se`;
```

Comment:
I think we will need to mark this as breaking.

Reply:
Done